requirements.txt

```diff
@@ -0,0 +1,6 @@
+mlflow>=2.0.0rc0
+pandas>=1.3.*
+scikit-learn>=1.1.*
+ipykernel>=6.12.*
+ipython>=7.32.*
+shap>=0.40.*
```

Comment:
I think pandas, sklearn and shap are no more needed: https://github.com/mlflow/recipes-regression-template/blob/main/requirements.txt

They are included in mlflow directly.